docs/functionality/ipc_tinker.md

```diff
@@ -0,0 +1,45 @@
+# IPC slot value tinker
+
+Each ipc slot offers a dbus api interface.
+One for filters, get value and value tinkering.
+
+## General information
+The slot requests a name from dbus `<slot_name>._slot_`
+storing its dbus connection, 
+The implemented functionality inject their interfaces
+into the slots established dbus connection.
+
+### Interface names
+the following interface names are used for the functionality.
+
+- filters `<slot_name>/filters`
```

Comment:
This might be prone to change but I will see how two bypass "config" in interface name within confman